### PR TITLE
Make selinuxd images configurable

### PR DESCRIPTION
**Pull request by mikroskeem**
#### What type of PR is this?

/kind feature

#### What this PR does / why we need it:

Makes selinuxd images configurable (in Helm chart)

One reason would be mirroring images on own registry, and pulling images from there.

However, while deploying SPO on Fedora CoreOS I found out that default image is not working for my deployment. This makes images configurable in more convenient manner.

See [discussion on Slack](https://kubernetes.slack.com/archives/C013FQNB0A2/p1718011082988209)

#### Which issue(s) this PR fixes:

None

#### Does this PR have test?

N/A

#### Does this PR introduce a user-facing change?

```release-note
Make selinuxd images configurable in Helm chart
```

### CODE REVIEW

File: deploy/helm/values.yaml [hunk 1]
```diff
@@ -6,6 +6,20 @@ spoImage:
   repository: k8s-staging-sp-operator/security-profiles-operator
   tag: latest
 
+selinuxdImage:
+  default:
+    registry: quay.io
+    repository: security-profiles-operator/selinuxd
+    tag: latest
```
Comment (saschagrunert) [inline deploy/helm/values.yaml:13]: We can also add security-profiles-operator/selinuxd-fedora :)

**Review by saschagrunert — CHANGES_REQUESTED** [review]

**Comment by saschagrunert** [comment]
Thank you!

/ok-to-test

File: deploy/helm/values.yaml (hunk 1 above)
Comment (mikroskeem) [reply deploy/helm/values.yaml:13]: There's no environment variable which reads this however

Comment (mikroskeem) [reply deploy/helm/values.yaml:13]: Perhaps this will do? fd182ac7b5c7b3d6907eab5280d5015d8d8657e7

Comment (saschagrunert) [reply deploy/helm/values.yaml:13]: Yep, thanks!

**Review by saschagrunert — APPROVED** [review]

**Review by saschagrunert — APPROVED** [review]

**Pull request merged**